modules/ROOT/pages/index-policies.adoc

```diff
@@ -1,8 +1,25 @@
-= Security Policies
+= Edge Security Policies
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
 
-Anypoint Security features a high-performance, reliable, and scalable service to enforce policies that apply to all nodes deployed to your Runtime Fabric. +
+Anypoint Security features a high-performance, reliable, and scalable service to enforce security policies that apply to all nodes deployed to your Runtime Fabric. 
 
-You can protect your APIs using these policies:
+== Prerequisites
+
+To configure and use the security policies, you must:
+
+. Have these permissions: 
+ ** API Manager *Manage Policies*
```

Comment:
Why is there text before the permission titles, which I assume are Manage Policies and Manage Runtime Fabrics?
(I would not boldface these.)